### PR TITLE
[FIX] mass_mailing: don't autopublish mailing lists

### PR DESCRIPTION
**Pull request by eapa-odoo**
Purpose:
- Changed the default value of 'Show In Preference' (is_public) field from true to false for preventing the automatic publishing of mailing lists.

Task-3594678

### CODE REVIEW

File: addons/mass_mailing/models/mailing_list.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class MassMailingList(models.Model):
         string='Subscription Information',
         copy=True, depends=['contact_ids'])
     is_public = fields.Boolean(
-        string='Show In Preferences', default=True,
+        string='Show In Preferences', default=False,
         help='The mailing list can be accessible by recipients in the subscription '
              'management page to allows them to update their preferences.')
```
Comment (awa-odoo) [inline addons/mass_mailing/models/mailing_list.py:43]: While we're at it, can we modify this help to remove the "s" of "to allows" ?

You will have to export the POT file as well to add the new entry.

**Review by awa-odoo — COMMENTED** [review]
Hello @eapa-odoo 

Looking good, just one minor comment to fix a pre-existing label.

Thanks & cheers!

File: addons/mass_mailing/models/mailing_list.py (hunk 1 above)
Comment (eapa-odoo) [reply addons/mass_mailing/models/mailing_list.py]: Hello, @awa-odoo
I have made the changes, can you please have a look?

Thanks :slightly_smiling_face:

**Comment by awa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**